File: Source/JapaneseLanguageTools.ReactSpa/source/models/blobs/responses/GetBlobMetadataResponse.Model.ts
import BlobMetadataModel from "@/models/blobs/BlobMetadata.Model";
import GetBlobMetadataRequestModel from "@/models/blobs/requests/GetBlobMetadataRequest.Model";

export default interface GetBlobMetadataResponseModel {
  blobMetadata: BlobMetadataModel;

  request: GetBlobMetadataRequestModel;
}

File: Source/JapaneseLanguageTools.Contracts/Models/Blobs/Responses/GetBlobMetadataResponse.Model.cs
using JapaneseLanguageTools.Contracts.Models.Blobs.Requests;

namespace JapaneseLanguageTools.Contracts.Models.Blobs.Responses;

public class GetBlobMetadataResponseModel
{
    public required BlobMetadataModel BlobMetadata { get; set; }

    public required GetBlobMetadataRequestModel Request { get; set; }
}
